internal/pkg/cli/testdata/autoscaling-svc-cf.yml

```diff
@@ -0,0 +1,462 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Nice! I'm trying to brainstorm how we can write better integ tests. Right now we just copy paste the output really of an existing deployment without checking its content :P

I think a nicer test would be having partial yaml (like only the autoscaling pieces of the template defined here) and making sure that the generated CloudFormation template contains that string. What do you think?

This would also make it less sensitive to custom resource code changes